astroplan/plots/time_dependent.py

```diff
@@ -187,6 +187,7 @@ def plot_airmass(targets, observer, time, ax=None, style_kwargs=None,
             (observer.sun_rise_time(Time(start), which='next').datetime, 0.1),
         ]
 
+        twilights.sort(key=lambda x: x[0])
```

Comment:
I generally prefer `operator.itemgetter(0)` over `lambda x: x[0]`. Since it's a standard builtin the additional import should be negligable.